js/lib/corejs-build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+npm i .
```

Comment:
This is failing for me when I run `build.sh`. `grunt` fails to install because of:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: core-js@2.5.1
npm ERR! Found: grunt@1.0.4
npm ERR! node_modules/grunt
npm ERR!   dev grunt@"1.0.x" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer grunt@"~0.4" from grunt-livescript@0.6.0
npm ERR! node_modules/grunt-livescript
npm ERR!   dev grunt-livescript@"0.6.x" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

`npm i --force` works, but maybe there's a better workaround.

Reply:
Hm ... that seems to be because I don't specify a version for node and it works with mine (14.x something) while it doesn't work with 15.4.0

Reply:
14.x is still the LTS, so as long as it works with that, I wouldn't bother. Usually, `current`, contains weird kinks until it reaches lts.